### PR TITLE
Google Analytics support

### PR DESCRIPTION
**Pull request by adrtod**
Add the Google Analytics support to your site by adding a tracking ID.
[Need help finding your tracking ID?](https://support.google.com/analytics/answer/1032385)

As suggested in #128, it is enabled via a key in `_pkgdown.yml`.

Example:

``` yaml
template:
  ganalytics: "UA-000000-01"
```

Undocumented at the moment. Let me know where to document if needed.

### CODE REVIEW

**Comment by hadley** [comment]
Thanks! I think the best place to document is probably in `build_site()` in the same block as `bootswatch`. When we get more parameters, we'll probably need to move elsewhere.

**Comment by adrtod** [comment]
Done!

File: R/build.r [hunk 1]
```diff
@@ -93,6 +93,17 @@
 #' See a complete list of themes and preview how they look at
 #' \url{https://gallery.shinyapps.io/117-shinythemes/}:
 #'
+#' Optionnally provide the \code{ganalytics} template parameter to enable
```
Comment (hadley) [inline R/build.r]: "Optionnally'

**Comment by adrtod** [comment]
:+1:

**Pull request merged**